.flutter-plugins-dependencies

```diff
@@ -0,0 +1 @@
+{"info":"This is a generated file; do not edit or check into version control.","plugins":{"ios":[{"name":"geolocator_apple","path":"/home/munsif/.pub-cache/hosted/pub.dev/geolocator_apple-2.3.2/","native_build":true,"dependencies":[]},{"name":"google_maps_flutter_ios","path":"/home/munsif/.pub-cache/hosted/pub.dev/google_maps_flutter_ios-2.3.3/","native_build":true,"dependencies":[]}],"android":[{"name":"flutter_plugin_android_lifecycle","path":"/home/munsif/.pub-cache/hosted/pub.dev/flutter_plugin_android_lifecycle-2.0.17/","native_build":true,"dependencies":[]},{"name":"geolocator_android","path":"/home/munsif/.pub-cache/hosted/pub.dev/geolocator_android-4.4.0/","native_build":true,"dependencies":[]},{"name":"google_maps_flutter_android","path":"/home/munsif/.pub-cache/hosted/pub.dev/google_maps_flutter_android-2.6.0/","native_build":true,"dependencies":["flutter_plugin_android_lifecycle"]}],"macos":[{"name":"geolocator_apple","path":"/home/munsif/.pub-cache/hosted/pub.dev/geolocator_apple-2.3.2/","native_build":true,"dependencies":[]}],"linux":[],"windows":[{"name":"geolocator_windows","path":"/home/munsif/.pub-cache/hosted/pub.dev/geolocator_windows-0.2.2/","native_build":true,"dependencies":[]}],"web":[{"name":"geolocator_web","path":"/home/munsif/.pub-cache/hosted/pub.dev/geolocator_web-2.2.0/","dependencies":[]},{"name":"google_maps_flutter_web","path":"/home/munsif/.pub-cache/hosted/pub.dev/google_maps_flutter_web-0.5.4+2/","dependencies":[]}]},"dependencyGraph":[{"name":"flutter_plugin_android_lifecycle","dependencies":[]},{"name":"geolocator","dependencies":["geolocator_android","geolocator_apple","geolocator_web","geolocator_windows"]},{"name":"geolocator_android","dependencies":[]},{"name":"geolocator_apple","dependencies":[]},{"name":"geolocator_web","dependencies":[]},{"name":"geolocator_windows","dependencies":[]},{"name":"google_maps_flutter","dependencies":["google_maps_flutter_android","google_maps_flutter_ios","google_maps_flutter_web"]},{"name":"google_maps_flutter_android","dependencies":["flutter_plugin_android_lifecycle"]},{"name":"google_maps_flutter_ios","dependencies":[]},{"name":"google_maps_flutter_web","dependencies":[]}],"date_created":"2023-12-21 09:22:27.324378","version":"3.16.4"}
```

Comment:
Please add this file to `.gitignore`

Reply:
sorry i forgot to add ! with the condition. 
it should be 
```dart
if(!context.mounted){
   return;
}
````

Reply:
> Please add this file to `.gitignore`

i done the changes can review it.